src/coord/rmr/chan.c

```diff
@@ -94,10 +65,8 @@ PushErrorMask MRChannel_Push(MRChannel *chan, void *ptr) {
     chan->head = chan->tail = item;
   }
   chan->size++;
-end:
-  if (pthread_cond_broadcast(&chan->cond)) rc |= BROADCAST_FAILURE;
```

Comment:
Why not have this check?

Reply:
why not check any other `pthread_*` call? I don't see a reason to check this one call in particular

Reply:
I think that if we want a pthread error handling, we should make a full effort and decide what to do for each call.
I don't see why this specific call should have this mechanism, but I removed the return code validation because we don't return an error code anymore